cogs/qrchecker/qrchecker.py

```diff
@@ -145,6 +169,43 @@ async def qrchecker(self, ctx: Context):
         """Configure QR code checker"""
         pass
 
+    @qrchecker.command("show")
+    async def show(self, ctx: Context):
+        """Show current settings"""
+        pixels = await self.config.get_attr(KEY_MAX_IMAGE_DIMENSIONS)()
+        enabled = await self.config.guild(ctx.guild).get_attr(KEY_ENABLED)()
+        msg = "**__Global settings:__**\n"
+        msg += f"Max pixels: **{'Unlimited' if not pixels else pixels} pixels**\n\n"
+        msg += "**__Guild settings:__**\n"
+        msg += f"Enabled: **{'Yes' if enabled else 'No'}**"
+        await ctx.send(msg)
+
+    @qrchecker.command("size")
+    async def maxSize(self, ctx: Context, *, pixels: Optional[int]):
+        """Set the maximum image pixels to check.
+
+        Binary images are loaded into RAM in Pillow, which uses RAM. If your bot is
+        running on a system with limited RAM, set this to a low value to avoid OOM
+        killer from killing your bot.
+
+        Parameters
+        ----------
+        size: Optional[int]
+            The maximum number of pixels in an image to check.
+        """
+        if pixels and pixels < 0:
+            await ctx.send("Please enter a positive number")
+            return
+        elif not pixels:
+            await ctx.send(
+                success("Disabled max pixels. The bot will check images of " "any size.")
```

Comment:
Is this a `black` thing?

```suggestion
                success("Disabled max pixels. The bot will check images of any size.")
```

Reply:
Sigh...yeah that was black